pkg/threescale/backend/manager.go

```diff
@@ -26,6 +27,7 @@ type Application struct {
 	LastResponse     *threescale.AuthorizeResult
 	LimitCounter     LimitCounter
 	UnlimitedCounter map[string]int
+	sync.RWMutex
```

Comment:
This rwlock is used to essentially protect modifications to the counters, but there is no similar lock (or reusing of this very lock) to protect setting LastResponse while it is being used in other places?

Reply:
👀